pkg/gui/services/custom_commands/handler_creator.go

```diff
@@ -80,8 +80,16 @@ func (self *HandlerCreator) call(customCommand config.CustomCommand) func() erro
 				f = func() error {
 					return self.menuPromptFromCommand(resolvedPrompt, wrappedF)
 				}
+			case "confirm":
+				f = func() error {
```

Comment:
can we chuck this in a `self.confirmPrompt` function for consistency with the cases?

Reply:
Definitely, that's a good idea. I moved it into the a separate `self.confirmPrompt` function: c8af4ad3a0e0adbcfc9da9cd8e40c19844fbcb27.

Note that I passed the handler function directly, instead of passing the `wrappedF` like other functions do. This is because the `types.ConfirmOpts.HandleConfirm` function has a different signature and `wrappedF` didn't seem necessary from what I can tell -- I could be mistaken 🙂.